packages/cache/src/index.tsx

```diff
@@ -19,32 +20,26 @@ import {
   incorrectImportAlarm
 } from './stylis-plugins'
 import { prefixer } from './prefixer'
-/* import type { StylisPlugin } from './types' */
+import { StylisPlugin } from './types'
 
-/*
-export type Options = {
-  nonce?: string,
-  stylisPlugins?: StylisPlugin[],
-  key: string,
-  container?: HTMLElement,
-  speedy?: boolean,
-  prepend?: boolean,
+export interface Options {
+  nonce?: string
+  stylisPlugins?: Array<StylisPlugin>
+  key: string
+  container?: Node
+  speedy?: boolean
+  /** @deprecate use `insertionPoint` instead */
+  prepend?: boolean
   insertionPoint?: HTMLElement
 }
-*/
 
 let getServerStylisCache = isBrowser
   ? undefined
-  : weakMemoize(() =>
-      memoize(() => {
-        let cache = {}
-        return name => cache[name]
```

Comment:
funny thing, I think this was incorrect since https://github.com/emotion-js/emotion/pull/1817 . It was working because we were able to store properties on this function that was returned from here. We should be returning an object though. The caller is responsible both for reading and writing to this "cache"